sdk/core/azure-core/src/main/java/module-info.java

```diff
@@ -41,6 +41,9 @@
     // Exports HttpProviders#getAllHttpClients API to azure-core-test module
     exports com.azure.core.implementation.http to com.azure.core.test;
 
+    // Exports UnixTime for testing.
+    exports com.azure.core.implementation to com.azure.core.test;
```

Comment:
I don't see any changes in `com.azure.core.test` module. Is this required?

Reply:
PR is a bit stale at this point, I'll get around to getting it up to date.